File: custom_helpers_py/get_paths.py
from dotenv import load_dotenv
from os.path import join
from os import environ

load_dotenv(override=True)


def get_out_folder() -> str:
    return environ.get("OUT_FOLDER")


def get_out_folder_house() -> str:
    return join(get_out_folder(), "house")


def get_out_folder_senate() -> str:
    return join(get_out_folder(), "senate")


def get_out_folder_analysis() -> str:
    return join(get_out_folder(), "analysis")

File: custom_helpers_js/getPaths.js
import "dotenv/config";
import { join } from "path";

export function getOutFolderHouse() {
  return join(getOutFolder(), "house");
}

export function getOutFolderSenate() {
  return join(getOutFolder(), "senate");
}

export function getOutFolderAnalysis() {
  return join(getOutFolder(), "analysis");
}

export function getOutFolderCapitolTrades() {
  return join(getOutFolder(), "capitol_trades");
}

export function getOutFolder() {
  return process.env.OUT_FOLDER;
}
